docs/integrations/app-development/opentelemetry/jira.md

```diff
@@ -0,0 +1,226 @@
+-   **Jira Webhooks**. Webhook events of types:
+    -   Issue
+    -   User
+    -   Sprint
+
+## Configure hosted collector to receive Webhooks
+
+In this step, you create a host collector to receive webhooks from Jira, and set up an HTTP source on it.
+
+1.  Configure a [hosted collector](https://help.sumologic.com/docs/send-data/hosted-collectors/configure-hosted-collector/), or select an existing hosted collector for the HTTP source.
+2.  Configure an [HTTP source](https://help.sumologic.com/docs/send-data/hosted-collectors/http-source/logs-metrics/) on the hosted collector.
+    -   For Source Category, specify Atlassian/Jira/Events
+    -   Add `sumo.datasource` as File to this source and assign the value **jira** for it.
+    <img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Jira-OpenTelemetry/Jira-Datasource.png')} alt="Datasource" />
+    -   Make a note of the HTTP address for the source. You will supply it when you configure a Jira webhook in the next step.
+
+## Register webhook in Jira
+
+
+Follow the instructions on [Webhooks](https://developer.atlassian.com/server/jira/platform/webhooks/) in Jira help to register a webhook for the following events:
+
+-   Issue related:
+    -   created (jira:issue_created)
+    -   updated (jira:issue_updated)
+    -   deleted (jira:issue_deleted)
+    -   worklog changed (jira:worklog_updated)
+-   User related:
+    -   created (user_created)
+    -   updated (user_updated)
+    -   deleted (user_deleted)
+-   Sprint related:
+    -   created (sprint_created)
+    -   deleted (sprint_deleted)
+    -   updated (sprint_updated)
+    -   started (sprint_started)
+    -   closed (sprint_closed)
+
+When you configure the webhook, enter the URL for the HTTP source you created in [step 2](https://help.sumologic.com/docs/integrations/app-development/jira/#Step_2:_Configure_hosted_collector_to_receive_webhooks) as the endpoint for the webhook.
+
+## Logs Collection Configuration & App installation
+
+As part of the setting up the collection process and app installation user can select the App from App Catalog and click on **Install App**. Please follow the steps below:
+
+### Step 1: Set up Collector
+
+If you want to create a new Collector please select **Add a new Collector** option.
+
+:::note
+If you want to use an existing OpenTelemetry Collector, you can skip this step by selecting the **Use an existing Collector** option.
+:::
+
+Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command which can be executed in the machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Jira-OpenTelemetry/Jira-Collector.png')} alt="Collector" />
+
+### Step 2: Configure integration
```

Comment:
```suggestion
### Step 2: Configure Integration
```

Reply:
I'd recommend using lowercase letters in the heading only if there are fewer than four letters.